corehq/apps/export/views/new.py

```diff
@@ -134,7 +136,7 @@ def page_context(self):
             sharing_options = [SharingOption.EDIT_AND_EXPORT]
 
         allow_deid = has_privilege(self.request, privileges.DEIDENTIFIED_DATA)
-
+        table_count = self.export_instance.get_default_selected_table_count()
```

Comment:
This line overwrites the values set on lines 123 and 125.

I'm not sure whether to right approach is to drop those lines, or this one.

(btw, does your IDE show `table_count` greyed out on lines 123 and 125?)

Reply:
That was a mistake on my end, I had it there originally and then realized that `table_count` should only be updated if it is not a bulk case export. I think I forgot that I had it there as well :sweat_smile: I have pushed a commit to remove the overriding line.

Mine does not show them greyed out. Hmm... I presume they are greyed out on PyCharm because those lines have no effect since the variable is getting overwritten further down.